src/docs/11_technical_risks.adoc

```diff
@@ -15,3 +15,11 @@ This should be your motto for systematic detection and evaluation of risks and t
 .Form
 List of risks and/or technical debts, probably including suggested measures to minimize, mitigate or avoid risks or reduce technical debts.
 ****
+
+Because of the complexity of this project and the use of technologies we have never tried before, the team may encounter some problems while developing the application. This are some of the problems we expect to have and, hopefully, surpass.
+
+. *Use of _React_:* No member of the team has ever worked with _React_ , so learning how this new language works will probably slow down the development.
+. *Use of _SOLID_:* None of us has ever worked with servers before and we had never heard of _SOLID_, so it is a scary and hard topic to tackle for all members of the team.
+. *Group work:* Because we are working in a group with a big number of members we may face some troubles when trying to communicate, organize and come to an agreement on some important decisions.
+. *Use of git:* Although this is not the first time we are using git we are still not experts on the subject (especially when using it from a command window), so some branch merges and conflicts may be difficult to perform and solve.
+. *Lack of experience:* As we are still third year students, there are a lot of libraries and helpful tools we don't know about, and some others that will be hard for us to use. Because of this lack of knowledge and experience, we may pass on using some unknown beneficial tools and, instead, waste our time trying to use more difficult ones.
```

Comment:
I would add the lack of time due to having other subjects to care about too, but seems fine to me